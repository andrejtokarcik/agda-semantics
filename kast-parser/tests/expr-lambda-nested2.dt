expr = (Lambda x Set 1 x) ((Lambda y Set 2 Set 0) Set 1)
