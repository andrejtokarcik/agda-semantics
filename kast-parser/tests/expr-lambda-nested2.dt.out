<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        expr |-> Set 1
    </tenv>
    <data>
        .Map
    </data>
    <fun>
        expr |-> Set 0
    </fun>
    <meta>
        .Map
    </meta>
</T>
